comps/llms/vllm-openvino/README.md

```diff
@@ -0,0 +1,67 @@
+# Use vLLM with OpenVINO
+
+## Build Docker Image
+
+To build the docker image, run the command
+
+```bash
+bash ./build_vllm_openvino.sh
+```
+
+Once it successfully finishes you will have a `vllm:openvino` image. It can directly spawn a serving container with OpenAI API endpoint or you can work with it interactively via bash shell.
+
+## Use vLLM serving with OpenAI API
+
+### Start The Server:
+
+```bash
+bash launch_model_server.sh
+```
+
+For gated models such as `LLAMA-2`, you will have to pass -e HUGGING_FACE_HUB_TOKEN=\<token\> to the docker run command above with a valid Hugging Face Hub read token.
+
+Please follow this link [huggingface token](https://huggingface.co/docs/hub/security-tokens) to get the access token and export `HUGGINGFACEHUB_API_TOKEN` environment with the token.
+
+```bash
+export HUGGINGFACEHUB_API_TOKEN=<token>
+```
+
+### Request Completion With Curl:
+
+```bash
+curl http://localhost:8000/v1/completions \
+  -H "Content-Type: application/json" \
+  -d '{
+  "model": "meta-llama/Llama-2-7b-hf",
+  "prompt": "What is the key advantage of Openvino framework?",
+  "max_tokens": 300,
+  "temperature": 0.7
+  }'
+```
+
+#### Customize vLLM-OpenVINO Service
+
+The `launch_model_server.sh` script accepts two parameters:
+
```

Comment:
Please add examples demonstrating the use of the two parameters. Developers indeed can and will look inside the launch script, but you want your readme to be more generally accessible. The docker run command is tucked in the launch script. Are you prescribing people use the launch script or directly run the docker run command. Unclear because you are providing instructions, intermingling both. Instead say use the script unless you want to customize aspects such as weights and Key Value (KV) compression. Weights compression particularly valuable for larger models. ..

Reply:
Added sample example in Readme and modified the launch script.